parent/pom.xml

```diff
@@ -99,7 +99,7 @@
 
     <!-- Dependency Versions -->
     <jackson.version>2.9.9</jackson.version>
-    <client-runtime.version>1.6.5</client-runtime.version>
+    <client-runtime.version>1.6.10</client-runtime.version>
```

Comment:
Did we publish this (1.6.10) version of azure-arm-client-runtime?

Reply:
yessir… yesterday

Reply:
thanks for confirming, LGTM